crates/flox/src/commands/environment.rs

```diff
@@ -563,7 +563,7 @@ impl Push {
                     bail!("Environment already linked to a remote")
                 }
 
-                Self::push_managed_env(&flox, managed_pointer, dir)
```

Comment:
suggestion blocking: we can remove the `#[allow(dead_code)] // not yet handled in impl` on line 550

Reply:
resolved [8608938](https://github.com/flox/flox/pull/449/commits/8608938cf8e95a8f899afafdd90fbac8f4ba129f)